common/init.sls

```diff
@@ -45,8 +45,15 @@ host-{{ hostname }}:
 {% endfor %}
 
 {% for ssh_user in common.ssh_users %}
+{{ ssh_user }}:
+  user.present:
+    - home: /home/{{ ssh_user }}
+    - optional_groups:
+        - wheel
+    - empty_password: True
```

Comment:
I'm ok with this for now (strictly better than status quo), but just wanted to leave a link to https://sourceforge.net/projects/pamsshagentauth/ as something I like to configure for password-less sudo that still has some security. I'm pretty sure there's nothing similar for OS X.